pkg/kube/pod_test.go

```diff
@@ -394,10 +395,17 @@ func (s *PodSuite) TestGetPodLogs(c *C) {
 		GenerateName: "test-",
 		Image:        consts.LatestKanisterToolsImage,
 		Command:      []string{"sh", "-c", "echo hello"},
+		PodOverride: crv1alpha1.JSONMap{
```

Comment:
Seems that this change does not belong to this PR.

Reply:
No, it belongs to PR - I add one more container to ensure that logs are always taken from the main container